<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitialWindow</class>
 <widget class="QMainWindow" name="InitialWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IR-tools</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="IRButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>260</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Run &quot;Reactor Imitator&quot;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ResultsPrcButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>260</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Results processing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PrepFilesButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>260</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Files preparation</string>
    </property>
   </widget>
   <widget class="QLabel" name="IRtoolsLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>300</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>48</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>IR-tools</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="HLine">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>420</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="GreetingsLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>300</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Greetings from the program</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ChooseOptionLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>380</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select the option you want to use:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="HappyUsingLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>300</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Happy using!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="Language">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="English"/>
     <addaction name="Russian"/>
    </widget>
    <addaction name="openHangingWindow"/>
    <addaction name="openFuelLoad"/>
    <addaction name="Language"/>
    <addaction name="separator"/>
    <addaction name="Exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">QStatusBar { 
					border-top-style: outset; 
					border-top-width: 1px; 
					border-top-color: #828282; 
					}</string>
   </property>
  </widget>
  <action name="English">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="Russian">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Russian</string>
   </property>
  </action>
  <action name="Exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="openHangingWindow">
   <property name="text">
    <string>Roll up into panel</string>
   </property>
  </action>
  <action name="openFuelLoad">
   <property name="text">
    <string>Open fuel load</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Exit</sender>
   <signal>triggered()</signal>
   <receiver>InitialWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
